File: CabsBooking.Web/KaiHungHuang.CabsBooking.Core/Entity/BookingHistory.cs
using System;

namespace KaiHungHuang.CabsBooking.Core.Entity
{
    public class BookingHistory
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public DateTime? BookingDate { get; set; }
        public string BookingTime { get; set; }
        public int? FromPlace { get; set; }
        public int? ToPlace { get; set; }
        public string PickupAddress { get; set; }
        public string Landmark { get; set; }
        public DateTime? PickupDate { get; set; }
        public string PickupTime { get; set; }
        public int? CabTypeId { get; set; }
        public string ContactNo { get; set; }
        public string Status { get; set; }
        public string Comp_time { get; set; }
        public decimal? Charge { get; set; }
        public string Feedback { get; set; }
        public Place FromPlaceName { get; set; }
        public Place ToPlaceName { get; set; }
        public CabType CabType { get; set; }
    }
}

File: CabsBooking.Web/ngCabsBooking/KaiHungHuang.CabsBookingSPA/src/app/core/shared/models/bookinghistory.ts
export interface BookingHistory {
    id?: number;
    email: string;
    bookingDate?: Date;
    bookingTime: string;
    fromPlace?: number;
    toPlace?: number;
    pickupAddress: string;
    landmark: string;
    pickupDate?: Date;
    pickupTime: string;
    cabTypeId?: number;
    contactNo: string;
    status: string;
    comp_time: string;
    charge?: number;
    feedback: string;
}
